src/exporter/ceph_exporter.cc

```diff
@@ -1,7 +1,7 @@
 #include "common/ceph_argparse.h"
 #include "common/config.h"
 #include "exporter/DaemonMetricCollector.h"
-#include "exporter/http_server.h"
+#include "exporter/web_server.h"
```

Comment:
I'm curious, why this change? HTTP is still a valid name even if you use HTTP + TLS. TLS is actually at the transport layer.

IMHO a "webserver" serves HTML formatted documents (plus CSS, Javascript and other assets), while Prometheus endpoints are just plain text HTTP servers.

Reply:
I leave it as [web server](https://en.wikipedia.org/wiki/Web_server)

Reply:
@jmolmo even if we finally decide with going with the name change I'm wondering why git is not able to track the name change here. It should be possible to see the changes yo have introduce but it seems like we are deleting a file and adding a new one. Is that the case? did you rename the file first before applying the patch?

Reply:
@rkachach. yes. I deleted the old file, not renamed. 
(I think probably because becauseI worked with web_server.cc without deleting the http_server.cc file ). And when I decided to put the pull request ,I removed all the old and not used files.

Reply:
Git usually is smart enough to track deletions + additions as file renames... In this case, the filename change might complicate tracking changes. @jmolmo , is there any practical reason to rename it?

Reply:
My fault @epuertat . It would be more easy the reviews if i did use the "mv" ... Taking into account time constraints, resources available (time), that the code is tested , and easy to "rebuild" the previous version. i am going to merge.
Thanks for pointing this .. next time i will be more careful!